### PR TITLE
PBI-13: video conference initial completion

### PR DESCRIPTION
**Pull request by rashadaziz**
image
image
image
image
image

### CODE REVIEW

**Comment by indiradr** [comment]
Screen Shot 2022-10-09 at 15 07 48
Hi Rashad! I think it would be better if the logos when the user wants to turn off their camera and microphone to be crossed out, like the logo at the corner of the screen as seen here:
Screen Shot 2022-10-09 at 15 08 28

It would provide better clarity for the users instead of just changing colors, keeping in mind for users who are color blind.

**Comment by rashadaziz** [comment]
> Screen Shot 2022-10-09 at 15 07 48
> 
> Hi Rashad! I think it would be better if the logos when the user wants to turn off their camera and microphone to be crossed out, like the logo at the corner of the screen as seen here:
> Screen Shot 2022-10-09 at 15 08 28
> 
> It would provide better clarity for the users instead of just changing colors, keeping in mind for users who are color blind.

Okay indi! thanks for the feedback, I'll work on those changes right now!

File: components/features/Dashboard/Applications/VideoConference/VideoPlayer/Settings/SettingsModal/SettingsModal.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { useEffect, useState } from "react";
+import { createPortal } from "react-dom";
+
+const DeviceSettings = () => {
+    return <></>
```
Comment (whenwe177) [inline components/features/Dashboard/Applications/VideoConference/VideoPlayer/Settings/SettingsModal/SettingsModal.tsx:5]: If I may ask shad, what will this component contain later?

File: services/Dashboard/VideoConference/index.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const getRoomToken = (userId: string) => {
+  return {
+    token: process.env.NEXT_PUBLIC_HMS_DEV_TOKEN,
```
Comment (whenwe177) [inline services/Dashboard/VideoConference/index.ts:3]: After creating the ODI account, remind me to share the token to everyone shad.

Comment (rashadaziz) [reply services/Dashboard/VideoConference/index.ts:3]: This token will be dynamically generated by our backend in production Alex. This token is just there for testing purposes

File: components/features/Dashboard/Applications/VideoConference/VideoPlayer/Settings/SettingsModal/SettingsModal.tsx (hunk 1 above)
Comment (rashadaziz) [reply components/features/Dashboard/Applications/VideoConference/VideoPlayer/Settings/SettingsModal/SettingsModal.tsx:5]: image
The idea of this component is more or less like this screenshot Alex. In the case where the user have malfunctioning devices, he/she may switch to a new one

**Pull request merged**